plugins/scaffolder-react/src/next/components/Form/DescriptionFieldTemplate.tsx

```diff
@@ -24,17 +24,17 @@ import {
   StrictRJSFSchema,
 } from '@rjsf/utils';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   markdownDescription: {
-    fontSize: '0.75rem',
+    fontSize: theme.typography.caption.fontSize,
```

Comment:
Oh woops - this is what you get for copying and pasting stuff that we have internally been experimenting with :joy:

Reply:
Easy to fix here. But finding the right override in our instance was a bit of a pain.

Learnt a lot about the styling in the 1.15 upgrade 😆